src/helpers/product.js

```diff
@@ -16,8 +16,27 @@
  */
 import cockpit from "cockpit";
 
+const getLastItemFromConsoleOutput = (text) => {
+    return text.split(" ").slice(-1)[0].replace("\n", "");
+};
+
 export const getAnacondaVersion = () => {
     return cockpit
             .spawn(["anaconda", "--version"])
-            .then((content) => content.split(" ").slice(-1)[0].replace("\n", ""));
+            .then(getLastItemFromConsoleOutput);
+};
+
+export const getAnacondaUIVersion = () => {
+    return cockpit
+            .spawn(["cockpit-bridge", "--packages"])
+            .then((content) => {
+                const pathToAnacondaWebUIAssets = content
+                        .split("\n")
+                        .filter(line => line.includes("anaconda-webui"))
+                        .map(getLastItemFromConsoleOutput)[0];
+                return cockpit
+                        .file(`${pathToAnacondaWebUIAssets}/VERSION.txt`)
+                        .read()
+                        .then((versionContent) => versionContent ? versionContent.replace("\n", "") : "N/A");
+            });
```

Comment:
Initially I wanted to collect this data using `rpm -qi anaconda-webui`, but the testing image don't have the rpm package (`bash: line 1: rpm: command not found` is what I get). Maybe that's just a side effect of the 

That's the reason I added this weird workaround to collect it.

Reply:
As for where to best read the Anaconda version from - I guess we can just dump it into some file & have it read from that via cockpit bridge - which it already does to a degree. :)